### PR TITLE
make compatible with swarm from master

### PR DESCRIPTION
**Pull request by ndk**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -4,7 +4,7 @@ Priority: extra
 Maintainer: Andrey Sibiryov <kobolog@yandex-team.ru>
 Build-Depends: cmake, cdbs, debhelper (>= 7.0.13), libcocaine-dev (>= 0.11.0.0),
  libboost-filesystem-dev, libboost-thread-dev, 
- libswarm-dev (>= 0.6.1.0), libswarm-dev (<< 0.7.0),
+ libswarm-dev (>= 0.6.1.0), libswarm-dev (<< 0.7.4),
```
Comment (ndk) [inline debian/control]: mb "libswarm-dev (>= 0.7.0), libswarm-dev (<< 0.7.4)"?

Comment (3Hren) [inline debian/control]: I think `libswarm-dev (>= 0.7.0)` should be enough, at least until there are no new swarm versions appeared.

**Comment by Kobolog** [comment]
@3Hren: should we merge this?

**Comment by 3Hren** [comment]
I'd like to ask @shindo for urlfetcher's support status first.

**Comment by 3Hren** [comment]
Thanks!

**Comment by faucct** [comment]
Should not this be merged into other branches too?

**Pull request merged**